source/common/tracing/http_tracer_impl.cc

```diff
@@ -168,18 +168,23 @@ HttpTracerImpl::HttpTracerImpl(DriverPtr&& driver, const LocalInfo::LocalInfo& l
 
 SpanPtr HttpTracerImpl::startSpan(const Config& config, Http::HeaderMap& request_headers,
                                   const Http::AccessLog::RequestInfo& request_info) {
-  std::string span_name = HttpTracerUtility::toString(config.operationName());
+  std::string span_name;
 
   if (config.operationName() == OperationName::Egress) {
-    span_name.append(" ");
-    span_name.append(request_headers.Host()->value().c_str());
+    span_name = request_headers.Host()->value().c_str();
```

Comment:
These changes reflect what is used within zipkin - i.e. it ignored the span name passed in, and instead just used the host header.

Doesn't appear that the lightstep tracer is specifically using the Egress/Ingress values, so must just be passed through to the backend tracing system - so we might need to double check with them that they are ok with these changed (default) values.

Reply:
cc @bhs